### PR TITLE
GSoC Julia jet reconstruction

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
GSoC project propsal, prepared with @hegner.

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
The link checker failure seems to be a false positive - the browser and via curl it's ok

File: _gsocproposals/2022/proposal_Julia-jet-reconstruction.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+The evaluation task comprises of doing a simple timing of the current anti-kt
+algorithm. For details please contact the mentors.
+
+If the candidate has prior experience in Julia they may submit this work as part
+of their evaluation.
+
+## Mentors
+
+* [Benedikt Hegner](mailto:Benedikt.Hegner@cern.ch) (CERN)
```
Comment (agheata) [inline _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:83]: Main mentor needs to be emphasized, and all mentors should be listed in `gsoc/2022/mentors.md`

File: _gsocproposals/2022/proposal_Julia-jet-reconstruction.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+mentor_avail: June-July, September-October
+organization:
+    - CERN
+---
+
+# Description
+
+<div style="text-align:center; padding:25px; float:right">
+<img src="{{ '/images/others/julia.png' | relative_url }}" alt="Julia Logo" width=100px />
+</div>
```
Comment (agheata) [inline _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:17]: The images are not found in my local checkout, although the files are there. Maybe just a feature in the local jekyll test?
Screenshot 2022-02-15 at 15 34 14
:

**Review by agheata — CHANGES_REQUESTED** [review]

File: _gsocproposals/2022/proposal_Julia-jet-reconstruction.md (hunk 2 above)
Comment (agheata) [reply _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:17]: Trying to replace with: `Julia Logo
` gives the same, basically `{{site.baseurl}}` is not expanded. Probably due to how the page is included by `_layouts/gsoc_proposal.html`?

Comment (aflp91) [reply _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:17]: Same thing on my laptop …

Comment (aflp91) [reply _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:17]: In fact, it's the `width=` that is wrong … The only alternative solution I found is to remove it and to resize the Julia logo.

Comment (agheata) [reply _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:17]: OK, the fix is to just put the width values between double quotes.

File: _gsocproposals/2022/proposal_Julia-jet-reconstruction.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+duration: 350
+mentor_avail: June-July, September-October
+organization:
+    - CERN
+---
+
+# Description
+
+<div style="text-align:center; padding:25px; float:right">
+<img src="{{ '/images/others/julia.png' | relative_url }}" alt="Julia Logo" width=100px />
```
Comment (agheata) [inline _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:16]:
```suggestion
<img src="{{ '/images/others/julia.png' | relative_url }}" alt="Julia Logo" width="100px" />
```

File: _gsocproposals/2022/proposal_Julia-jet-reconstruction.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+([Julia in HEP Paper][JuliaHEPPaper], [Julia HEP Github][JHEPGH]).
+
+In order to continue this investigation, this project will re-implement some
+important algorithms for HEP in Julia, with the aim of assessing developer
+convenience and runtime performance compared to standard implementations in C++.
+
+## Jet Reconstruction
+
+<div style="text-align:center; padding:25px; float:right">
+<img src="{{ '/images/others/ATLAS_VP1_event_display_run282712_evt474587238_2015-10-21T06-26-57_v3.png' | relative_url }}" alt="ATLAS Event Display" width=400px />
```
Comment (agheata) [inline _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:35]:
```suggestion
<img src="{{ '/images/others/ATLAS_VP1_event_display_run282712_evt474587238_2015-10-21T06-26-57_v3.png' | relative_url }}" alt="ATLAS Event Display" width="400px" />
```

File: _gsocproposals/2022/proposal_Julia-jet-reconstruction.md (hunk 1 above)
Comment (agheata) [inline _gsocproposals/2022/proposal_Julia-jet-reconstruction.md:83]:
```suggestion
* **[Benedikt Hegner](mailto:Benedikt.Hegner@cern.ch)** (CERN)
```

**Review by agheata — CHANGES_REQUESTED** [review]

**Review by agheata — APPROVED** [review]

**Pull request merged**